src/app/Models/Complaint.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Complaint extends Model
+{
+    use HasFactory;
+
+    /**
+     * @var array
+     */
+    protected $fillable = [
+        'reviewed_by',
+        'reviewed_at',
+        'is_fixed',
+        'result',
+        'subject',
+        'description',
+        'related_user_id',
+        'created_by'
+    ];
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function createdUser()
+    {
+        return $this->belongsTo(User::class, 'created_by');
+    }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function reviewedUser()
+    {
+        return $this->belongsTo(User::class, 'reviewed_by');
+    }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function relatedUser()
```

Comment:
methodlara return type ekleyebiliriz,  hayat kurtarıcı olabiliyorlar

```suggestion
    /**
     * @return BelongsTo
     */
    public function relatedUser(): BelongsTo
```